### DISCUSSION TITLE
Docker Hub Image with latest Elsa 3.0

### DISCUSSION THREAD

**Discussion by markjones-WK** [elsa-workflows/elsa-core#4663] [Q&A]
@sfmskywalker, today, I can't seem to get the latest v3 docker hub image working.  Is there an issue with this one that you are aware of?  

Do you use the Elsa.AllInOne.Web project to build this image?

**Comment by sfmskywalker** [comment]
I haven't tested it since the targeting of .NET 8, so it's possible that there's an issue with it. I'll check it out. Thanks for letting me know!

> Do you use the Elsa.AllInOne.Web project to build this image?

Yep

**Reply by sfmskywalker** [reply]
I just tried downloading & running a fresh copy of the latest image, with no issues. Please make sure to delete the current image before trying to pull the latest one; Docker for Desktop tends to cache the current image, presumably because it only looks at the :latest tag (but I'm not sure if that's the case; I would expect it to compare some hash).

**Reply by markjones-WK** [reply]
Interesting.  I just tried again from two different laptops and docker and podman (which was a new installation) with same results.  Even tried with incognito mode.

This is what I am getting in the browser windows:
![image](https://github.com/elsa-workflows/elsa-core/assets/126109777/1581c475-51c0-45c5-946f-0e72133007da)

Nothing is shown in the container logs and all I see in the console is this:
![image](https://github.com/elsa-workflows/elsa-core/assets/126109777/9778f6ca-ad28-4779-a64e-a8487ba9dbe6)

I am checking with other co-workers as well.

**Reply by navoznov** [reply]
![image](https://github.com/elsa-workflows/elsa-core/assets/1484295/a746521c-19ad-4ca5-a1b2-a0d55e53769f)

Almost the same for me.

**Reply by sfmskywalker** [reply]
Do you have steps to reproduce this? What are the exact shell commands you used?

**Comment by markjones-WK** [comment]
FYI, I am still getting this error using the latest v3 docker hub image.  I have tried on 3 different laptops so I doubt it is cached related.

Any ideas?

**Comment by markjones-WK** [comment]
I figured it out.  Apparently, the image changed which port is was listening on internally.  It changed from 80 to 8080.  So, I changed the command to:

docker run -t -i -e ASPNETCORE_ENVIRONMENT=Development -p 13000:8080 elsaworkflows/elsa-v3:latest.

This needs to be update in the documentation:

In the 'Running the docker image' section at: https://v3.elsaworkflows.io/docs/installation/docker
